README.md

```diff
@@ -96,6 +89,19 @@ The configuration file requires a simple configuration in the following format:
 useInstancePrincipals: true
 ```
 
+#### Driver Kubernetes API Access
+
+The driver needs to get node information from the Kubernetes API server. A kubeconfig file with appropriate permissions (rbac: nodes/get) needs
+to be provided in the same manor as the OCI auth config file above.
+
+```
+kubectl create secret generic oci-flexvolume-driver-kubeconfig \
+    -n kube-system \
+    --from-file=kubeconfig=kubeconfig
+```
+
+Once the Secret is set and the daemonsets deployed, the kubeconfig file will be placed onto the master nodes.
```

Comment:
Once the Secret is set and the DaemonSet is deployed, the kubeconfig file will be placed onto the master nodes.